Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -796,9 +796,7 @@ class BrowserViewController: UIViewController,
 
         // During split screen launching on iPad, this callback gets fired before viewDidLoad gets a chance to
         // set things up. Make sure to only update the toolbar state if the view is ready for it.
-        // Ecosia: Update check
-        // if isViewLoaded {
-        if isViewLoaded && UIDevice.current.userInterfaceIdiom == .pad {
```

Comment:
@d4r1091 Am I missing some context on why this was introduced as part of https://github.com/ecosia/ios-browser/pull/712? Do you see an issue with reverting that and using it like previously (and also like what firefox currently has)?

Reply:
@lucaschifino I've been trying to find out the root cause of having that check in. I've also been trying to play with the app landscape/portrait with the original Firefox check (same as your revert proposal  here) and could not find anything strange. 
Happy to revert it to its original, and less of Ecosia's stuff here 💪